boomkat_importer.user.js

```diff
@@ -60,7 +57,12 @@ function getLinkType(media) {
     return MBImport.URL_TYPES.purchase_for_mail_order;
 }
 
-function retrieveReleaseInfo(release_url) {
+async function fetchTracksAndInsertLink(release_url, element) {
```

Comment:
Please give `element` a more descriptive name like `formatElement` (or `format_element`), otherwise it is hard to follow the purpose of this value across multiple function calls.

Reply:
Good point. Fixed.